drivers/docker/fingerprint.go

```diff
@@ -137,6 +138,19 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 
 		fp.Attributes["driver.docker.runtimes"] = pstructs.NewStringAttribute(
 			strings.Join(runtimeNames, ","))
+		fp.Attributes["driver.docker.os_type"] = pstructs.NewStringAttribute(dockerInfo.OSType)
+
+		if runtime.GOOS == "windows" && dockerInfo.OSType != "windows" {
```

Comment:
Where does the OSType string come from? Is there a const we can use or a link to docs we can put in a comment?

My concern is that if we get this wrong we disable Docker on Windows completely.

Reply:
I have updated the check to check for Linux containers specifically to avoid this risk and added a test to ensure that docker reports healthy on Windows.

OSType is basically `runtime.GOOS` of the docker daemon - https://github.com/moby/moby/blob/dd945557876ba90c1b2c2eb6af17daf1dcdc83e3/pkg/platform/platform.go#L22